x-pack/legacy/plugins/apm/public/components/app/ServiceNodeMetrics/index.tsx

```diff
@@ -47,17 +47,22 @@ export function ServiceNodeMetrics() {
 
   const { data: { host, containerId } = INITIAL_DATA, status } = useFetcher(
     callApmApi => {
-      if (serviceName && serviceNodeName) {
+      if (serviceName && serviceNodeName && start && end) {
         return callApmApi({
           pathname:
             '/api/apm/services/{serviceName}/node/{serviceNodeName}/metadata',
           params: {
-            path: { serviceName, serviceNodeName }
+            path: { serviceName, serviceNodeName },
+            query: {
+              start,
+              end,
+              uiFilters: JSON.stringify(uiFilters)
+            }
           }
         });
       }
     },
-    [serviceName, serviceNodeName]
+    [end, serviceName, serviceNodeName, start, uiFilters]
```

Comment:
For consistency should we keep the arguments order the same?
```suggestion
    [serviceName, serviceNodeName, start, end, uiFilters]
```